src/main/java/data/Camera.java

```diff
@@ -0,0 +1,24 @@
+package data;
+
+        import lombok.Getter;
+        import lombok.Setter;
+
+/**
+ * Created by Bart.
+ * Class to store information about cameras.
+ */
+public class Camera {
+
+    // Description of the camera, things like serial number.
+    @Getter
+    private String description;
+
+    // Number of counts this camera needs at maximum to move to a new position.
```

Comment:
This comment doesn't really help me with understanding what the variable exactly does. What is this 'margin'? Where do the counts come from? The description doesn't really match the variable name.